pyscriptjs/src/main.ts

```diff
@@ -192,9 +192,10 @@ export class PyScriptApp {
 
         this.logStatus('Initializing web components...');
         // lifecycle (8)
-        createCustomElements(interpreter);
 
-        initHandlers(interpreter);
+        //Takes a runtime and a reference to the PyScriptApp (to access plugins)
+        createCustomElements(runtime, this);
+        initHandlers(runtime);
```

Comment:
Since we don't use `runtime` perhaps this needs to be updated?

```suggestion
        //Takes an interpreter and a reference to the PyScriptApp (to access plugins)
        createCustomElements(interpreter, this);
        initHandlers(interpreter);
```

Reply:
Thank you!